src/connectors/voice-search/connectVoiceSearch.ts

```diff
@@ -74,10 +74,8 @@ const connectVoiceSearch: VoiceSearchConnector = (
     const { searchAsYouSpeak = false } = widgetParams;
 
     return {
-      getConfiguration() {
-        return {
-          query: '',
-        };
+      getConfiguration(config) {
+        return config.setQuery(config.query || '');
```

Comment:
The behaviour with the previous value should be covered by a test. IMO it's better to implement the "previous" behaviour in a separate PR. We can do a pass on each widget at the same time.